.gitignore

```diff
@@ -1,5 +1,6 @@
 *.jl.*.cov
 *.jl.cov
 *.jl.mem
+*.Identifier
```

Comment:
I don't know what a `.Identifier` file is but I have never seen or heard of them, so I doubt it should go into this `.gitignore`.

Reply:
I think it is some mac thing I am fine with having it in the git ignore.

Reply:
but @dstarkenburg might consider adding it to his global gitignore anyway.

Reply:
It might have to do with me using Windows Subsystem for Linux, I will add it to my global and remove it from the package.